citus.spec

```diff
@@ -45,24 +46,32 @@ make %{?_smp_mflags}
 %{__cp} README.md %{buildroot}%{pginstdir}/doc/extension/README-%{sname}.md
 %{__cp} NOTICE %{buildroot}%{pginstdir}/doc/extension/NOTICE-%{sname}
 
-# Set paths to be packaged other than LICENSE, README & CHANGELOG.md
+# Set paths to be packaged for citus  other than LICENSE, README & CHANGELOG.md
 echo %{pginstdir}/include/server/citus_*.h >> installation_files.list
 echo %{pginstdir}/include/server/distributed/*.h >> installation_files.list
 echo %{pginstdir}/lib/%{sname}.so >> installation_files.list
 echo %{pginstdir}/share/extension/%{sname}-*.sql >> installation_files.list
 echo %{pginstdir}/share/extension/%{sname}.control >> installation_files.list
+
 %ifarch ppc64 ppc64le
   %else
   %if 0%{?rhel} && 0%{?rhel} <= 6
   %else
+    # citus file paths to be added into package
     echo %{pginstdir}/lib/bitcode/%{sname}/*.bc >> installation_files.list
     echo %{pginstdir}/lib/bitcode/%{sname}*.bc >> installation_files.list
     echo %{pginstdir}/lib/bitcode/%{sname}/*/*.bc >> installation_files.list
-    
+
     # Columnar does not exist in Citus versions < 10.0
     # At this point, we don't have %{pginstdir},
     # so first check build directory for columnar.
     [[ -d %{buildroot}%{pginstdir}/lib/bitcode/columnar/ ]] && echo %{pginstdir}/lib/bitcode/columnar/*.bc >> installation_files.list
+    # Columnar files moved into its own directory like a seperate extension. Lines below adds these files into
+    # package file list
+    [[ -f %{buildroot}%{pginstdir}/lib/%{sname_columnar}.so  ]] && echo %{pginstdir}/lib/%{sname_columnar}.so >> installation_files.list
```

Comment:
Instead of having 4 long lines, can you write an if block?

If `citus_columnar.so` file exists, we expect all the directories to contain some bytecode files. IMO it is enough to check if the `.so` file is moved to a new directory.